### DISCUSSION TITLE
v2.11.5

### DISCUSSION THREAD

**Discussion by mienaiyami** [mienaiyami/yomikiru#166] [General]
## Downloads

[![Download setup](https://img.shields.io/badge/Windows%20Setup%20(exe)-Yomikiru--2.11.5--Setup.exe-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.11.5/Yomikiru-2.11.5-Setup.exe)

[![Download Portable](https://img.shields.io/badge/Windows%20Portable%20(zip)-Yomikiru--win32--2.11.5--Portable.zip-brightgreen?logo=windows&logoColor=blue)](https://github.com/mienaiyami/yomikiru/releases/download/v2.11.5/Yomikiru-win32-2.11.5-Portable.zip)

---
## Note

Linux distribution currently not available as direct download but can be build easily by following this [Build for linux(debian) guide](https://github.com/mienaiyami/yomikiru/blob/master/docs/build_for_linux.md).

---

## Added

- Option to disable Hardware Acceleration.

## Changed

- UI Enhancements.
- Switched from web based fullscreen to electron fullscreen.
- Smoother sidelist width change.
- In "Open chapter directly under base directory...", allowed to update list instead of "no image" error.

## Fixed

- Multiple scrolling issues in LTR/RTL mode with vertically fit enabled.
- System buttons showing when opening file from file explorer with zen mode. #147.

---

#### Todo

- Write tests.
- Add ability to change image quality in canvas mode for less RAM usage and performance.

---

**Full Changelog**: https://github.com/mienaiyami/yomikiru/compare/v2.11.4...v2.11.5

<hr /><em>This discussion was created from the release <a href='https://github.com/mienaiyami/yomikiru/releases/tag/v2.11.5'>v2.11.5</a>.</em>

**Comment by ranenvious** [comment]
In regards to making a linux binary, WSL (windows subsystem for linux) should allow compilation to a linux binary on windows. For inter-distro compatibility though Appimage or Flatpaks would probably be preferable however since then a single binary can be used on every distro.
Not a developer myself so I unfortunately can't help with actually packaging it though.

**Reply by mienaiyami** [reply]
It does but thats not the reason. I dont have a server so I build windows executable on my pc and upload it with some automation. But if I were to build for linux as well I would have to switch to wsl and update code to latest then build for linux and upload manually.

Im thinking up setting up some github action to build for linux automatically but Im busy for a week.
